git_machete/github.py

```diff
@@ -116,7 +113,11 @@ def __fire_github_api_request(method: str, url: str, token: Optional[str], reque
         headers['Authorization'] = 'Bearer ' + token
 
     host = 'https://api.' + GITHUB_DOMAIN
-    http_request = Request(host + url, headers=headers, data=json.dumps(request_body).encode(), method=method.upper())
+    url = host + path
+    json_body: Optional[str] = json.dumps(request_body) if request_body else None
+    http_request = Request(url, headers=headers, data=json_body.encode() if json_body else None, method=method.upper())
```

Comment:
A little fix: previously, if `None` was passed as `request_body`, `data` would be the string `"null"` (as this is a JSON value corresponding to Python `None`) rather than `None`